test/asyncJest.spec.js

```diff
@@ -10,13 +10,11 @@ a função recebe como parâmetro true e false, respectivamente.
 ATENÇÃO!!! Edite apenas este arquivo. Não altere os arquivos da pasta 'src'.
 */
 
-describe('1 - O retorno do telefonema', () => {
-  test('atende', () => {
-    assert.fail();
-    // Insira seu teste assíncrono aqui
+describe('Requisito 1 - O retorno do telefonema', () => {
```

Comment:
```suggestion
describe('1 - O retorno do telefonema', () => {
```

Fala @gustavogss ! Vi que você adicionou a palavra requisito aqui, isso faz o nosso avaliador não reconhecer o teste. Pode remover, como sugiro?

Reply:
Valeu, Isaac, eu pensei que não ia afetar em nada isso